nextflow_schema.json

```diff
@@ -162,12 +165,12 @@
             "properties": {
                 "multiqc_title": {
                     "type": "string",
-                    "default": null,
+                    "default": "None",
```

Comment:
Same question as before, while the previous version sets this to be empty effectively, this sets it to "None". I'd like to make sure these changes are intentional.

Reply:
Same as comment above - I just want it to be where the user is required to enter a lineage with no defaults. I'm not sure if this null or None in this case?

Reply:
Should be turned back to null